airflow/cli/commands/celery_command.py

```diff
@@ -37,7 +37,7 @@
 WORKER_PROCESS_NAME = "worker"
 
 
-@cli_utils.action_logging
+@cli_utils.action_cli()
```

Comment:
Is there a reason why we've got the empty parens on all these?

Reply:
FWIW I personally like to do this because I hate the _first argument is the function unless you need a flag when the decorator becomes a function_ complexity, and perfer to always just implement the callable-returning-decorator case.